migrations/versions/0317_add_provider_response.py

```diff
@@ -0,0 +1,21 @@
+"""
+
+Revision ID: 0317_add_provider_response
+Revises: 0316_now_live_template
+Create Date: 2021-04-14 10:00:42.383782
+
+"""
+
+revision = '0317_add_provider_response'
+down_revision = '0316_now_live_template'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('notifications', sa.Column('provider_response', sa.Text(), nullable=True))
```

Comment:
About the choice of `text`

> There is no performance difference among these three types, apart from increased storage space when using the blank-padded type, and a few extra CPU cycles to check the length when storing into a length-constrained column. While character(n) has performance advantages in some other database systems, there is no such advantage in PostgreSQL; in fact character(n) is usually the slowest of the three because of its additional storage costs. In most situations text or character varying should be used instead.

https://www.postgresql.org/docs/11/datatype-character.html